dirs/dirs.go

```diff
@@ -88,6 +89,7 @@ func SetRootDir(rootdir string) {
 	SnapdSocket = filepath.Join(rootdir, "/run/snapd.socket")
 
 	SnapBinariesDir = filepath.Join(SnapAppsDir, "bin")
+	SnapDesktopDir = filepath.Join(rootdir, "/usr/share/applications")
```

Comment:
Yeah, same problem on snappy. This is branch is intended for the desktop, I guess we will want something better here for the phone and snappy, probably a new XDG_DATA_DIR that points to /apps/ (?). I'm not sure this is a blocker at this point though.